### PR TITLE
Add header into backup file

### PR DESCRIPTION
**Pull request by msoap**

### CODE REVIEW

File: cmd/tabledumper/tabledumper.go [hunk 1]
```diff
@@ -72,7 +74,7 @@ func main() {
 	}
 	log.Printf("Will use %d streams", cfg.Streams)
 	dd := dir_dumper.
-		NewDirDumper(cfg.Dir, dbInfo).
+		NewDirDumper(cfg.Dir, cfg.WithHeader, dbInfo).
```
Comment (dmitry-vovk) [inline cmd/tabledumper/tabledumper.go]:
```go
NewDirDumper(cfg.Dir, dbInfo).
WriteHeader(cfg.WithHeader).
```

File: table_dumper/dumper.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func NewTableDumper(dsn, tableName string, config Config) *Dumper {
 	}
 }
 
-func (d *Dumper) Run(w io.Writer, conn *sqlx.DB) (stats, error) {
+func (d *Dumper) Run(w io.Writer, conn *sqlx.DB, withHeader bool) (stats, error) {
```
Comment (dmitry-vovk) [inline table_dumper/dumper.go]: Move `withHeader` parameter into `config`

File: cmd/tabledumper/tabledumper.go (hunk 1 above)
Comment (msoap) [reply cmd/tabledumper/tabledumper.go]: done.

File: table_dumper/dumper.go (hunk 2 above)
Comment (msoap) [reply table_dumper/dumper.go]: Added as new private field

**Pull request merged**